alea/examples/configs/unbinned_wimp_statistical_model.yaml

```diff
@@ -63,7 +66,7 @@ parameter_definition:
   er_band_shift:
     nominal_value: 0
     ptype: shape
-    uncertainty: null  # stats.uniform(loc=-2, scale=4)
+    uncertainty: 'stats.uniform(loc=-2, scale=4)'
```

Comment:
Not sure if a uniform constraint makes much sense here tbh

Reply:
Are `null` and `'stats.uniform(loc=-2, scale=4)'` equivalent?

Reply:
No. null won't add an uncertainty, it sets it to None, which is the default value. I had this commented out to later be used once I would implement the str-uncertainty but I didn't so far. So maybe we can just go back to null and remove the comment

Reply:
Yes, this discussion will converge on the discussion of str-uncertainty.